data/generate.fsx

```diff
@@ -344,6 +348,10 @@ File.Delete(csvFilePath)
 
 printfn "Creating table %A as CSV" customers.Name
 
+// reset the RNG, and be careful about the same order of generated tables,
```

Comment:
I find these comments a little overly verbose.

Reply:
Indeed, they are verbose :(. But this still beats missing this and testing on different data, which I did for a while :) . I think the TODO above, to make the generation of SQLite/CSV consistently seeded, is the proper way to resolve this. Or, ideally, one should be able to export CSV from SQLite, but I couldn't find a way which would play nicely with this generator. Maybe I could re-approach this later, with a fresh mind.